MTM-53127 Added icon selector info to the docs. (#1459)
SoftwareAG/c8y-docs: content/standard-tenant/ecosystem-bundle/application-properties.md

<td style="text-align:left">Automatically created</td>
<td style="text-align:left">Specified by the user; for example, if you use "hello" as application path, the URL of the application will be "/apps/hello"</td>
</tr>
<tr>

> <td style="text-align:left">Select Icon</td>

Reviewer: ```suggestion
<td style="text-align:left">Select icon</td>
```

> <td style="text-align:left">Opens a modal in which a wide variety of icons can be selected.</td>

Reviewer: ```suggestion
<td style="text-align:left">Provides a variety of icons from which an icon for the application can be selected.</td>
```
